drivers/virtio/9p/Config.uk

```diff
@@ -0,0 +1,9 @@
+config VIRTIO_9P
```

Comment:
After a brief discussion with @skuenzer: Please update all `Config.uk` options in virtio to from `VIRTIO_` to `LIBVIRTIO_`. Remember to update your Makefiles and anywhere else where these maybe be used.